src/pages/ViewInterviewPage.tsx

```diff
@@ -94,6 +94,9 @@ const InterviewView = ({ interview }: { interview: HydratedInterview }) => {
         <ModalContent maxH="800px" maxW="700px">
           <ModalCloseButton />
           <ReportPopUp
+            title="Report Interview"
+            badgeColorScheme="red"
```

Comment:
It looks like all are red? We should just default to red and allow the option to pass in different colours if needed.

Reply:
yes can add default

Reply:
oh alr set as default, will remove